database/migrations/2024_03_18_012142_rename_realops_est_arr_time_to_est_time_enroute.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void {
+        DB::table("realops_flights")->truncate();
```

Comment:
Is there something forcing us to truncate the table? I'm concerned that this will wipe our test data - we're so close to the event we may not have time to rebuild...

Reply:
If we don't do this, the data in the table will suddenly become garbage.
Take this for an example (some liberties taken for the example ofc):
Before:
| Callsign | Departs | Arrives |
|------------|------------|----------|
| AAL233 | 12:35 | 14:35 |

After:

| Callsign | Departs | Enroute Time |
|------------|------------|----------|
| AAL233 | 12:35 | 14h 35m |

Export the existing database to CSV and DM it to me. I'll write up a script to convert and generate a sql file you can import back into the database to fix this, but I talked with Ian and this was his suggestion.

Reply:
@iccowan I care less about having an accurate ETE for testing... IMO it's more important that we have data to test with. I'm reluctant to truncate our test data because we're so close to the event. I'd be in favor of either not truncating the table and accepting that the arrival time will get turned into a duration for testing or having the migration set every field to an arbitrary test value (ex. 1:11). I don't really want to create more work for Tyler or us exporting/importing data. Do you have any other thoughts on this?

Reply:
I didn't consider test data when making that recommendation. I think it would be best to just truncate the table to prevent bad data in prod (as long as Meg is aware of it), but if that's going to create a bunch of extra work I'm fine to just to swap the name as long as no types are changing (and again make sure Meg is aware so she can adjust the data on prod).